optimum/exporters/tasks.py

```diff
@@ -192,54 +213,53 @@ class TasksManager:
         "image-segmentation": "transformers",
         "masked-im": "transformers",
         "semantic-segmentation": "transformers",
-        "speech2seq-lm": "transformers",
-        "audio-ctc": "transformers",
+        "automatic-speech-recognition": "transformers",
         "audio-classification": "transformers",
         "audio-frame-classification": "transformers",
         "audio-xvector": "transformers",
-        "vision2seq-lm": "transformers",
+        "image-to-text": "transformers",
         "stable-diffusion": "diffusers",
         "zero-shot-image-classification": "transformers",
         "zero-shot-object-detection": "transformers",
     }
 
-    # TODO: some models here support causal-lm export but are not supported in ORTModelForCausalLM
+    # TODO: some models here support text-generation export but are not supported in ORTModelForCausalLM
     # Set of model topologies we support associated to the tasks supported by each topology and the factory
     _SUPPORTED_MODEL_TYPE = {
         "audio-spectrogram-transformer": supported_tasks_mapping(
-            "default",
+            "feature-extraction",
             "audio-classification",
             onnx="ASTOnnxConfig",
         ),
         "albert": supported_tasks_mapping(
```

Comment:
I would make it so that `supported_tasks_mapping` can accept synonyms as well. But not primordial either.

Reply:
Good call, that'd be interesting if external users are writing their own mappings. Is that something that happens frequently?